scripts/checks/coverage.sh

```diff
@@ -4,11 +4,8 @@ set -euo pipefail
 
 export COVERAGE=true
 export FOUNDRY_FUZZ_RUNS=10
-export NODE_OPTIONS="${NODE_OPTIONS:-}"
 
-if [[ $NODE_OPTIONS != *"--max-old-space-size"* ]]; then
-  export NODE_OPTIONS="${NODE_OPTIONS} --max-old-space-size=8192"
-fi
+eval "scripts/set-max-old-space-size.sh"
```

Comment:
do we need an eval here ?
```suggestion
scripts/set-max-old-space-size.sh
```

Reply:
This was already updated the line is now
```
. ./scripts/set-max-old-space-size.sh
```